source/code-snippets/usage-examples/changeStream.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  ChangeStream,
+  MongoClient,
+} from "mongodb";
+
+// Replace the uri string with your MongoDB deployment's connection string.
+const uri =
+  "mongodb+srv://<user>:<password>@<cluster-url>?writeConcern=majority";
+
+const client = new MongoClient(uri);
+
+async function run(): Promise<void> {
```

Comment:
We don't think that the TS example in this case provides any additional value to the JS example

Reply:
Adding fix in [PR with similar comment](https://github.com/mongodb/docs-node/pull/233#discussion_r686873680). Although this example will soon be changed, I think it is still worth documenting the lack of TS specific features for readers and the person who picks up the ticket to remake this example.

Reply:
Agreed